Cargo.toml

```diff
@@ -273,6 +274,7 @@ members = [
   "components/concurrency_manager",
   "components/file_system",
   "components/collections",
+  "components/coprv2_plugin_api",
```

Comment:
I prefer `coprocessor_plugin_api`, what do you think? @skyzh

Reply:
Agree.

Reply:
Alright, renamed to `coprocessor_plugin_api` :)